test/model_test.dart

```diff
@@ -1493,29 +1493,41 @@ String topLevelFunction(int param1, bool param2, Cool coolBeans,
 
   group('Typedef', () {
     Typedef t;
```

Comment:
A minor nit - this should have been named better than `t` originally, but now that we have a 2nd related variable, perhaps a longer name  (`typedef`?)